.kci-dev.toml.example

```diff
@@ -2,20 +2,15 @@ default_instance="local"
 
 [local]
 host="https://127.0.0.1"
+api="https://127.0.0.1:8001/"
```

Comment:
I don't see why change host with API.
I think is confusing with what need to be write, as to what API actually refer too as it could refer to version or API system and not only host address